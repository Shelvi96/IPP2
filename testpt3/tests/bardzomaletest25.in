NEW ID

;6;:81014>65;1325284
7641515>91
302;675>4;7743995
5154>:1;33047
9:>35351651:
:>989:5183
;69>256;
42>005:561
9469>426491
7:69>49
@36130;0:097974:4:5876428:443
@:44550980792369;81:5293640
@88;16793090928:438528
@42125;69765;74647;600802
@7:19:0205864270813799;8
@980502170
@9333
@9;15
@955280;:37:1
@;681086359:84;;6547:99;1814
@;5724844
@2;139979852404197298:9044200:
