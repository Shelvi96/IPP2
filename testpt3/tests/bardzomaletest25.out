2576821041303552
17894590564608
71914346
20101000000
10355665836
0
0
0
0
69045681164207
0
8354501269085565
